### PR TITLE
feature: analytics basic version + menu button in left navigation + settings in hamburger menu nav

### PR DESCRIPTION
**Pull request by tomaszrondio**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -126,12 +124,38 @@ export default angular.module('superdesk-publisher', [
...
             template: require('./client/views/settings/index.html'),
             sideTemplateUrl: 'scripts/apps/workspace/views/workspace-sidenav.html',
         });
+
+        workspaceMenuProvider.item({
+            href: '/publisher/dashboard',
+            label: gettext('Publisher'),
+            icon: 'publisher',
+            order: 1100,
+            group: 'Planning'
```
Comment (ahilles107) [inline index.js]: this group is correct?

File: client/views/analytics/index.html [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+                                <a href="" class="sd-content-nav__btn"
+                                    ng-click="webPublisherAnalytics.setTenant(site)"
+                                    ng-class="{'sd-content-nav__btn--active' : webPublisherAnalytics.selectedTenant.code === site.code}">
+                                    {{:: site.name }}</a>
+                            </li>
+                        </ul>
+                    </nav>
+                </div>
+                <div class="sd-column-box__main-column sd-column-box__main-column--flex">
+                   <!-- <div class="subnav subnav--lower-z-index">
```
Comment (ahilles107) [inline client/views/analytics/index.html]: do we need to merge that commented code?

File: index.js (hunk 1 above)
Comment (tomaszrondio) [reply index.js]: yes

File: client/views/analytics/index.html (hunk 2 above)
Comment (tomaszrondio) [reply client/views/analytics/index.html]: yes

**Pull request merged**